### ISSUE TITLE
Searching in JEI Crash

### ISSUE THREAD

**Issue by WenXin20** [UniversalLP/JustEnoughButtons#20]
1.10.2-Forge 2221
JEI v3.14.5.406
JEB v1.6.5
I got a crash while typing "oak" into the JEI search bar.
[Crash Report](http://pastebin.com/bNBduQw8)
[fml-client-latest.txt](https://github.com/UniversalLP/JustEnoughButtons/files/769648/fml-client-latest.txt)

**Comment by univrsal** [comment]
Ok, so I cam across this issue myself when testing a while back.
For me it was because of charset storage, which caused a crash when JEI tried to display some of the items the mod adds. 
But since you don't have charset installed it has to be another mod causing this, because neither JEI nor JEB crash when I type "oak" without any other mods installed.

JEI had a similiar [issue](https://github.com/mezz/JustEnoughItems/issues/261), which couldn't be solved since it was caused by another mod.

TL;DR I'm not sure what's causing this, nor how to fix it :P, but my first question would be: Does searching for "oak" crash without JustEnoughButtons installed?

**Comment by WenXin20** [comment]
I haven't had a chance to try without JEB yet, but I noticed that it happened after I added [Project Red-World](https://minecraft.curseforge.com/projects/project-red-world), which requires [Project Red-Base](https://minecraft.curseforge.com/projects/project-red-base), [MrTJPCore](https://minecraft.curseforge.com/projects/mrtjpcore), &[ Forge MultiPart CBE](https://minecraft.curseforge.com/projects/forge-multipart-cbe) which requires [Code Chicken Lib 1.8+](https://minecraft.curseforge.com/projects/codechicken-lib-1-8).

**Comment by univrsal** [comment]
Hmm, I installed all Project Red related mods and I can't reproduce the issue

**Comment by WenXin20** [comment]
It still happened without JEB. Any ideas what could be the cause?

**Comment by univrsal** [comment]
Well, from what I found out I think it's and item or a block from a mod that has a custom renderer which crashes when JEI tries to draw it in the item list.
What does the crashlog say this time?
But I think your best bet is to ask someone like [mezz](https://github.com/mezz/) since he's working on JEI and generally way more capable than I am.

**Comment by univrsal** [comment]
Alright I'll just go ahead and close this, since it's nothing on my side

**Comment by WenXin20** [comment]
Ok, thanks for the help.
